src/services/fileStorage/strategies/interface.js

```diff
@@ -0,0 +1,17 @@
+class AbstractFileStorageStrategy {
+	constructor() {
+		if (new.target === AbstractFileStorageStrategy) {
+			throw new TypeError("Cannot construct AbstractFileStorageStrategy instances directly.");
+		}
+	}
+
+	create() {
+		throw new TypeError("create method has to be implemented.");
+	}
+
+	getFiles() {
+		throw new TypeError("getFiles method has to be implemented.");
+	}
+}
```

Comment:
das war nicht in dem ticket mit enthalten. deleting haben wir sowieso komplett erstmal rausgenommen, weil aws da ziemliche einschränkungen macht